tests/func/test_checkout.py

```diff
@@ -56,6 +56,35 @@ def _test_checkout(self):
         self.assertTrue(filecmp.cmp(self.FOO, self.orig, shallow=False))
 
 
+class TestCheckoutExecutable(TestRepro):
+
+    EXECUTABLE = "executable"
+    EXECUTABLE_CONTENT = "executable content"
+
+    def setUp(self):
+        super().setUp()
+
+        self.create_executable(self.EXECUTABLE, self.EXECUTABLE_CONTENT)
```

Comment:
The only place where `create_executable` is used, better to inline it here too :)

Btw, we are migrating from unittest to pytest tests, so it would be great to re-write this one with pytest too.

Reply:
Inlined `created_executable` here too.